src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -26,7 +27,7 @@ beforeEach(() => {
   jest.spyOn(Router, 'useParams').mockReturnValue({
     id: '123',
   });
-  jest.spyOn(Router, 'useHistory').mockReturnValue({});
+  jest.spyOn(Router, 'useHistory').mockReturnValue({} as any);
```

Comment:
Are these mocks needed? Maybe you could wrap component with MemoryRouter by using render function from testUtils

Reply:
I will refactor these to use the new render helper method